### DISCUSSION TITLE
No Parameters of DJT11LM (Aquara/ Xiomi)

### DISCUSSION THREAD

**Discussion by ak68-hub** [Koenkk/zigbee2mqtt#18918] [Devices]
I´m using HA at Raspberry Pi 4 with Sonoff Zigbee 3.0 USB Dongle Plus V2 (the "new" one !)

Pairing of DJT11LM (Aquara) with Sonof and HA was possible 
and I see all of the entitates, 
but I can´t see any parameters in HA: all of them are " unknown"

How do I get the parameters ????
Thank you very much :)

PS: I DO get Parameter from other Aquara-sonsors like Water-Leck or Windows-Sensors

**Comment by juhacz** [comment]
Hi 
I have the same problem, I only see linkquality value.

{
    "linkquality": 255
}

Did you solve the problem?
Aquara DJT11LM
Core 2024.3.3
Supervisor 2024.03.1
Operating System 12.1
Interfejs użytkownika 20240307.0
Zigbee2mqtt: : 1.36.0-1
